------------------------------------------------------------
revno: 14152
revision-id: squid3@treenet.co.nz-20170331233921-efxhs8vy025fvrnl
parent: squid3@treenet.co.nz-20170331233831-m3hfrigo82uhz4id
committer: Amos Jeffries <squid3@treenet.co.nz>
branch nick: 3.5
timestamp: Sat 2017-04-01 12:39:21 +1300
message:
  libtrie: Fix 'make check' when run before 'make all'
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squid3@treenet.co.nz-20170331233921-efxhs8vy025fvrnl
# target_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# testament_sha1: 8399bbfe7b517fa6306bdc61d212a9a4fcc9e88b
# timestamp: 2017-03-31 23:40:52 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# base_revision_id: squid3@treenet.co.nz-20170331233831-\
#   m3hfrigo82uhz4id
# 
# Begin patch
=== modified file 'lib/libTrie/Makefile.am'
--- lib/libTrie/Makefile.am	2017-01-01 00:16:45 +0000
+++ lib/libTrie/Makefile.am	2017-03-31 23:39:21 +0000
@@ -8,8 +8,8 @@
 include $(top_srcdir)/src/Common.am
 include $(top_srcdir)/src/TestHeaders.am
 
-DIST_SUBDIRS = test
-SUBDIRS = test
+DIST_SUBDIRS = . test
+SUBDIRS = . test
 
 noinst_LIBRARIES = libTrie.a
 

